staging/src/k8s.io/client-go/dynamic/client_test.go

```diff
@@ -34,6 +34,9 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	restclient "k8s.io/client-go/rest"
 	restclientwatch "k8s.io/client-go/rest/watch"
+
+	// install to get v1 fallbacks for Get/List/DeleteOptions registered
+	_ "k8s.io/client-go/pkg/api/install"
```

Comment:
This is installing to api.Scheme. The VersionedParameterEncoderWithV1Fallback is using the client-go/kubernetes/scheme.ParameterCodec. So i don't thinks this install is effective.

Please do now add more dependency on client-go/pkg/api. It's the package for internal types and will be gone after the [refactor](https://github.com/kubernetes/kubernetes/pull/44784) i'm working on :)